tests/test_switch.py

```
@@ -540,7 +550,14 @@ async def test_unbound_and_vpn_variations(coordinator, ph_hass, make_config_entr
     created: list = []
 
     async def run_setup():
+        """Run the public setup flow and collect the created switch entities."""
+
         def add_entities(ents):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `add_entities`

Add return type annotation: `None`

(ANN202)